src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
 body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: widthCalendar($cell-size, $dayGap, $padding);
+  padding: 30px $padding;
+  gap: $dayGap;
+  margin: 0 auto;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cell-size;
+    height: $cell-size;
+    background-color: $cell-color;
+    border: 1px solid $border-color;
+    transition: 0.5s;
```

Comment:
Let's specify the property for which we want to add transition duration. If you don't specify it - it will affect in the bad way on the website performance